### ISSUE TITLE
[jcomm] Need to change my email at community.joomla.org

### ISSUE THREAD

**Issue by toivo** [joomla/joomla-websites#1934]
The request from a newly registered user was posted to the Joomla! Forum: [Need to change my email at community.joomla.org](https://forum.joomla.org/viewtopic.php?f=7&t=1004931)

Thank you!

Toivo Talikka
Joomla! Forum, Global Moderator

**Comment by mgscreativa** [comment]
Hi! I'm the one with the issue, thanks @toivo for taking the time to report this. I will explain the issue I'm facing.

I registered my account at https://community.joomla.org/ but I wrote my email address wrong, hence, I can't email verify my account. My account username is mgscreativa and the wrong email is martin@mgscreativa.om (mispelled the .com).

I know I can create a new account with different username and correct email but I want to keep my account username.

Who should I contact for support on changing my account email at community.joomla.org?

Thanks!
